core/modules/user/user.module

```diff
@@ -1442,6 +1442,16 @@ function user_menu() {
     'access arguments' => array('administer permissions'),
     'file' => 'user.admin.inc',
   );
+  $items['admin/config/people/roles/add'] = array(
+    'title' => 'Add role',
```

Comment:
The page title doesnt seem to inherit this title still just says "Roles"

Reply:
Hmm true. Not sure why that is. Let's discuss in the issue.